USAGE.md

```diff
@@ -102,7 +122,7 @@ class CustomParser < Packwerk::Parsers::Erb
   end
 end
 
-Packwerk::Parsers::Factory.instance.erb_parser_class = CustomParser
+Packwerk::Parsers::Factory.instance.parsers = [Packwerk::Parsers::Ruby, CustomParser]
 ```
```

Comment:
I noticed that we can use `Packwerk::FileParser.remove` for simpler rewriting.

```ruby
Packwerk::FileParser.remove(Packwerk::Parsers::Erb)

class CustomErbParser < Packwerk::Parsers::Erb
  include Packwerk::FileParser

  def parse_buffer(buffer, file_path:)
    preprocessed_source = buffer.source

    # Comment out <%graphql ... %> tags. They won't contain any object
    # references anyways.
    preprocessed_source = preprocessed_source.gsub(/<%graphql/, "<%#")

    preprocessed_buffer = Parser::Source::Buffer.new(file_path)
    preprocessed_buffer.source = preprocessed_source
    super(preprocessed_buffer, file_path: file_path)
  end
end
```

It is a bit strange to inherit from `Packwerk::Parsers::Erb` and include `Packwerk::FileParser`, but it seems unavoidable. You might be able to get around it by using `Class#inherited`, but I'm not sure.

This method is not specific to the ERB parser and can be applied to all parsers.
Thus,  a more appropriate section title might be "How to override a parser" rather than "Using a custom ERB parser".